.gitignore

```diff
@@ -10,3 +10,4 @@
 
 # rspec failure tracking
 .rspec_status
+.DS_Store
```

Comment:
I disagree here. there is no need for storing .DS_Store files ever.

Reply:
I agree, but there's also no need for a lot of different files (*.swp, *~, etc...)  If we want to add every file type that we don't committed for every OS/Editor/IDE this list will get enormous.  It is on us as reviewers to make sure that people aren't adding files that they shouldn't as part of the PR review process.

Reply:
Actually this just came up in the ManageIQ community recently https://github.com/ManageIQ/manageiq-ui-classic/pull/7305#issuecomment-691489962

Reply:
This is under the IBM namespace not ManageIQ, which means we can decide our policy for ourselves. I have no problem with a large gitignore file if it keeps the repo clean of cruft and saves from ever having those files checked-in in the first place. Besides I think the handwaving of it'll be large is a bit overblown, we can start with the files we know about now and if someone else comes along with additional requirements we can add those at that time. This really just fixes the problem once and for all.

Reply:
I'll weight in here. It may seem scrupulous to reject a small addition to the gitignore, but in general we're going to be better able to maintain this repo if we keep the content strictly relevant to the IBM Cloud SDK. In this case, adding .DS_Store to your global gitignore accomplishes everything you need _and_ gives you autonomy to make changes without community review.

Reply:
I don't understand this discussion. DS_Store files are never relevant to a repo. It seems backwards to allow someone to check them in and then have a discussion about not checking them in when you can prevent it at the source.

Reply:
I don't want to hold up this PR while we debate different gitignore philosophies, adding `.DS_Store` to gitignore isn't related to adding support for the VPC SDK and thus should be a separate PR